dockerImages.yml

```diff
@@ -8,4 +8,7 @@
 
 # Tools are sorted alphabetically for ease of use.
 
-gatk4: "quay.io/biocontainers/gatk4:4.1.2.0--1"
\ No newline at end of file
+"gatk4": "quay.io/biocontainers/gatk4:4.1.2.0--1"
```

Comment:
What is the difference between setting the names for the images in "" and not doing so, since we don't use the "" in a lot of the other pipelines?

Reply:
This is a `Map[String, String]`. Strings are in quotes. This is the same in other pipelines. It might be different in the YAML files, but these don't require quoting strings.